tasks/oraInst/checkOraInst.yml

```diff
@@ -15,37 +15,37 @@
 # It is recommended that this file (oraInst.loc) be created in /etc directory
 - name: "Create {{ ora_inst }}"
   template:
-    src: "oraInst.loc.j2"
-    dest: "{{ ora_inst }}"
+    src: 'oraInst.loc.j2'
+    dest: '{{ ora_inst }}'
   become: True
   when: not ora_file.stat.exists
 
 - name: "Create oinstall group"
   group:
-    name: "{{ install_group }}"
+    name: '{{ install_group }}'
     state: present
   become: True
   when: not ora_file.stat.exists
 
 - name: "Add {{ ohs_user }} to {{ install_group }}"
   user:
-    name: "{{ ohs_user }}"
-    groups: "{{ install_group }}"
+    name: '{{ ohs_user }}'
+    groups: '{{ install_group }}'
     append: True
   become: True
 
 - name: "Check inventory directory"
   stat:
-    path: "{{ inventory_directory }}"
+    path: '{{ inventory_directory }}'
   register: inventory
```

Comment:
Why do we need explicit check?

Reply:
Use this task in all oracle roles ([weblogic](https://github.com/lean-delivery/ansible-role-weblogic/blob/master/tasks/ora-inst/check-ora-inst.yml#L32-L36), [oracledb](https://github.com/lean-delivery/ansible-role-oracle-db/blob/master/tasks/oraInst/checkOraInst.yml#L26-L29), etc)